Base/QTCore/qSlicerExtensionsManagerModel.cxx

```diff
@@ -1943,18 +1944,19 @@ bool qSlicerExtensionsManagerModel::installExtension(
     if (!unresolvedDependencies.isEmpty())
       {
       success = false;
-      qWarning() << tr("%1 extension depends on the following extensions, which could not be found: %2")
+      qWarning() << QString(/*no tr*/"%1 extension depends on the following extensions, which could not be found: %2")
         .arg(extensionName)
         .arg(unresolvedDependencies.join(", "));
       if (d->Interactive)
         {
-        QString msg = QString("<p>%1 depends on the following extensions, which could not be found:</p><ul>").arg(extensionName);
+        //: %1 is the extension name
+        QString msg = QString("<p>%1</p><ul>").arg(tr("%1 depends on the following extensions, which could not be found:")).arg(extensionName);
```

Comment:
This was also incorrect and now fixed in https://github.com/Slicer/Slicer/pull/6793

To avoid this type of parenthesis issue, I suggest to "split" in two lines:

```suggestion
        //: %1 is the extension name
        QString txt = tr("%1 depends on the following extensions, which could not be found:").arg(extensionName)
        QString msg = QString("<p>%1</p><ul>").arg(txt);
```

Reply:
Thanks @lassoan and @jcfr for your comments and fixes.

By the way, these last weeks, I have some issues making a local build of Slicer since I made a synchronization with the `Slicer/main` repository. I have errors on parts of the code I didn't touch.

Even the build of the non-edited main repository doesn't work. What prevents me from correctly testing my i18n updates.

The issue seems related to the recent updates made on the `qMRMLLayoutManager` class but I don't have yet a good knowledge of its logic to try to fix it directly.

```
103>D:\Slc\S5\Libs\MRML\Widgets\qMRMLLayoutManager.h(271,12): error C3668: 'qMRMLLayoutManager::createViewport': method with override specifier 'override' did not override any base class methods
103>D:\Slc\S5\Libs\MRML\Widgets\qMRMLLayoutManager.h(273,8): error C3668: 'qMRMLLayoutManager::onViewportUsageChanged': method with override specifier 'override' did not override any base class methods
```

I attached the log contents of `Output` and `Error List` tabs of my Visual Studio in case one may help.

Regards.

Reply:
Hi @mhdiop - did you also update CTK?  Usually it's enough to just rebuild the Slicer-build directory but sometimes there are updates outside of the Slicer source code that require rebuilding dependencies.  

You can keep an eye on this by watching for changes to the [SuperBuild](https://github.com/Slicer/Slicer/commits/main/SuperBuild) directory when updating from git since your last top-level build.

We can discuss more in our meeting, but I guess [this commit](https://github.com/Slicer/Slicer/commit/07d928e5a5a0934b0773c4c76762a7656973bd51) is the issue from about 2 weeks ago.

Reply:
Hi @pieper 

Thank your for your tips, I didn't think to update CTK.

Unfortunaly, I've done it but I still have the same problem.

Regards.

Reply:
1. Build the top-level Slicer.sln. This will update CTK source code. It is OK if this build fails. 
2. Build the top-level CTK.sln (CTK is not rebuilt by the top-level Slicer.sln)
3. Build the top-level Slicer.sln it should succeed.

Reply:
Thanks @lassoan for your feedback.

I tried it but it still doesn't work.

@pieper showed me another approach that I'm testing now. For reference, it's as follows :
1. Open directly the top-level `Slicer.sln` file on  Visual Studio (not by passing through the `Slicer.exe --VisualStudioProject` command
2. Build the `ALL_BUILD` project

I tried it and it seems to work. The build haven't finished yet but at least I see a different behaviour on the process.

Regards.

Reply:
Hi all.

Just to precise for reference that the last method proposed by @pieper worked.

Thanks all for your helps.

Regards.